src/tests/auth/test_auth_query.py

```diff
@@ -58,15 +58,17 @@ def test_set_catalog_collection(self):
         self.assertEqual(q._authority._collection, "any collection")
         self.assertEqual(q._authority._auth_scheme, GOB_AUTH_SCHEME)
 
-    @patch("gobapi.auth.auth_query.extract_roles")
-    def test_get_roles(self, mock_extract):
+    def test_get_roles(self):
         mock_req = MagicMock()
+        delattr(mock_req, 'roles')
```

Comment:
Niet echt nodig. Indien niet gezet zou worden zou bij de assertEqaal in line 68 == MagicMock.